src/C5 이진탐색/P02 JS3 떡볶이 떡 만들기.js

```diff
@@ -0,0 +1,31 @@
+function binarySearch(arr, target, start, end) {
+  let result = 0;
+  while (start <= end) {
+    let total = 0;
+    let mid = parseInt((start + end) / 2);
+    arr.forEach((val) => {
+      if (val > mid) {
+        total += val - mid;
+      }
+    });
```

Comment:
```suggestion
    const total = arr.reduce((acc, val) => acc + Math.max(0, val - mid), 0);
```

검증은 못해봤는데, reduce 로 줄여서 쓸 수도 있을거 같아요 :smile: